.github/workflows/ci.yml

```diff
@@ -41,11 +41,14 @@ jobs:
       matrix:
         os:
           - 'debian-8'
-          - 'debian-9'
```

Comment:
We should keep Debian 9 but remove Debian 8 (same in the other kitchen configs).